docs/development/managing-users/index.md

```diff
@@ -0,0 +1,55 @@
+# Managing users
+
+To manage users you will need an admin token and connection to the REANA-Server component:
+
+```console
+$ kubectl exec -ti reana-server-xxxx bash
+> read -s REANA_ADMIN_ACCESS_TOKEN
```

Comment:
The prompt `> ` may be misleading if it is too far from the `kubectl` statement... especially on later parts of the page.

We could use some more explicit prompt e.g. `root@pod$ ` instead, but perhaps the best would be to use a "closest-to-real-life" prompt example:

```
$ kubectl exec -i -t reana-server-aaaa-bbbb /bin/bash
...
root@reana-server-aaaa-bbbb:/code# reana-admin ...
```